projects/kubernetes-csi/external-attacher/Makefile

```diff
@@ -10,9 +10,23 @@ SOURCE_PATTERNS=./cmd/csi-attacher
 
 HAS_RELEASE_BRANCHES=true
 
+EXTRA_GO_LDFLAGS=-X main.version=$(GIT_TAG)
+
 include $(BASE_DIRECTORY)/Common.mk
 
 
+build: validate-cli-version
```

Comment:
This blob is a common pattern we have used in a few other [spots](https://github.com/aws/eks-distro/blob/main/projects/kubernetes-sigs/aws-iam-authenticator/Makefile#L25)